wrappers.json

```diff
@@ -0,0 +1,200 @@
+{
+    "ivy": {
+        "functional": ["negative.so",
```

Comment:
As discussed before, we shouldn't be hardcoding the names of the wrappers but instead have some way to pull those binaries based on whether they exist programmatically, as it becomes much more easier to manage the usage 😅

Reply:
The reason for this is the automation building the binaries can't be sure which ones are working (at least in its current state). So, there has to be some way to specify which ones are working correctly (identified with manual testing for now). We could move this to the `cython-wrappers` repo till we get it automated. Please let me know what you think. I do agree it'll be better to have only working binaries on the binaries repo.

Reply:
Yeah exactly, ig we can have some check to ensure that we'd only have the working binaries on the binaries repo 😅
Ofc even the current approach works but it isn't as scalable, so probably we can just assume that all binaries on the binaries repo are working for now. This would lead to a few failures with the cython wrappers mode, but we've got tests to identify that anyway, what do you think?

Reply:
One more problem 😅. I'm using the JSON file to make sure I only mark those functions that have a wrapper implementation with the `download_cython_wrappers_wrapper`. The alternative would be to wrap every function (but then we'd end up re-wrapping it with the Python wrappers once the download fails (for functions where the wrapper doesn't exist). As for scalability, I think for the time-being we can manually update the JSON file with working wrappers.. What do you think?

Reply:
Yeah let's do the manual json for now, but eventually we'd need the programmatic approach as you mentioned where we re-wrap with the python wrappers once the download fails. Thanks!